python/tests/api/writer/test_whylabs_integration.py

```diff
@@ -0,0 +1,162 @@
+import os
+import time
+from uuid import uuid4
+
+import pandas as pd
+import pytest
+from whylabs_client.api.dataset_profile_api import DatasetProfileApi
+from whylabs_client.model.profile_traces_response import ProfileTracesResponse
+
+import whylogs as why
+from whylogs.api.writer.whylabs import WhyLabsWriter
+from whylogs.core import DatasetProfileView
+from whylogs.core.feature_weights import FeatureWeights
+from whylogs.core.schema import DatasetSchema
+from whylogs.core.segmentation_partition import segment_on_column
+
+# TODO: These won't work well if multiple tests run concurrently
+
+os.environ["WHYLOGS_NO_ANALYTICS"] = "True"
+# WHYLABS_API_ENDPOINT, WHYLABS_API_KEY, WHYLABS_DEFAULT_ORG_ID, and
+# WHYLABS_DEFAULT_DATASET_ID need to come from the environment
+
+ORG_ID = os.environ["WHYLABS_DEFAULT_ORG_ID"]
+MODEL_ID = os.environ["WHYLABS_DEFAULT_DATASET_ID"]
```

Comment:
```suggestion
ORG_ID = os.environ.get("WHYLABS_DEFAULT_ORG_ID")
MODEL_ID = os.environ.get("WHYLABS_DEFAULT_DATASET_ID")
```

Reply:
It's probably easier to diagnose the missing key exception than the collateral damage of having these end up as `None`?

Reply:
The writers have ok exceptions, we just need to avoid reading the values outside of the tests in a way that throws.

Reply:
If you want you can move the read of the org/dataset ids into the test methods?